src/components/layout/TwoColumnsLayout.tsx

```diff
@@ -15,26 +15,39 @@ interface Props {
     url: string;
   };
   leftContent?: ReactNode;
+  leftContentColumns?: number;
```

Comment:
would it improve the interface if we would do something like:
```typescript
left: {
   content: ReactNode;
   columns: number;
}
```

Reply:
are the props really optional? IMO if the component is called `TwoColumnsLayout` it is clear that you must pass a left and right content

Reply:
We made it optional initially. I was mulling on the interface because that would be a breaking change, if we're doing it anyway we might as well make the props mandatory

Reply:
I'll leave that up to you, is fine for me how it is